manim/mobject/mobject.py

```diff
@@ -292,6 +293,7 @@ def apply_to_family(self, func):
         for mob in self.family_members_with_points():
             func(mob)
 
+    @accepts_unit
```

Comment:
Personally, I thought this was one of the easiest functions to add this functionality to. Once we are satisfied with how it works we should extend it to other functions.